server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -26,7 +26,8 @@ import {
 import { generateServiceProtocolEntries } from "@fluidframework/protocol-base";
 import { FileMode } from "@fluidframework/protocol-definitions";
 import { defaultHash, IGitManager } from "@fluidframework/server-services-client";
-import { Lumber, LumberEventName } from "@fluidframework/server-services-telemetry";
+// eslint-disable-next-line max-len
```

Comment:
Let's remove this eslint disable comment and instead break the import down into multiple lines?

```ts
import {
    Lumber,
    LumberEventName,
    Lumberjack,
    getLumberBaseProperties,
} from "@fluidframework/server-services-telemetry";
```